CONTRIBUTION_LADDER.md

```diff
@@ -0,0 +1,121 @@
+# Contribution Ladder
+
+---
+
+* [Roles](#roles)
+* [Community Member](#community-member)
+* [Contributor](#contributor)
+  * [How to become a contributor](#how-to-become-a-contributor)
+* [Maintainer](#maintainer)
+  * [How to become a maintainer](#how-to-become-a-maintainer)
+  * [Involuntary Removal or Demotion](#involuntary-removal-or-demotion)
+  * [Stepping Down/Emeritus Process](#stepping-downemeritus-process)
+---
+
+Our ladder defines the roles and responsibilities for this project and how to
+participate with the goal of moving from a user to a maintainer. You will need
+to gain people's trust, demonstrate your competence and understanding, and meet
+the requirements of the role.
+
+## Roles
+
+* Community Member
+* Contributor
+* Maintainer
+
+## Community Member
+
+Everyone is a community member! 😄 You've read this far so you are already ahead. 💯
+
+Here are some ideas for how you can be more involved and participate in the community:
+
+* Comment on an issue that you’re interested in.
+* Submit a pull request to fix an issue.
+* Report a bug.
+* Come chat with us in [Slack][slack].
+
+They must follow our [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
Note that a lot of the procedures for this are controlled by the GOVERNANCE doc in the OPA repo.